Changed const character in js file

MrKumaran opened the pull request:
Changed const characters from array of character to string for readablity

MrKumaran committed:
Update README.md

MrKumaran committed:
Update README.md

MrKumaran committed:
Update README.md

MrKumaran committed:
Changed characters:
Characters don't need to be array of characters makes reading tough it can be array will work just fine

MrKumaran committed:
Update README.md

ravikishore23 commented:
Yena arachi panuriga

On Sun, 27 Apr, 2025, 12:16 am Kumaran S, ***@***.***> wrote:

> Changed const characters from array of character to string for readablity
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1
> Commit Summary
>
>    - fe1c4eb
>    <https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1/commits/fe1c4ebdafd3a36bf750259333a45c7966beeaab>
>    Update README.md
>    - 5f2e16a
>    <https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1/commits/5f2e16af12c07160f87390192fa8b2fc77c8c234>
>    Update README.md
>    - 3905cd8
>    <https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1/commits/3905cd8c7fb34bdaa8eaf06c88da4774cd94a62c>
>    Update README.md
>    - 06916e4
>    <https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1/commits/06916e4c01b46a57cd4e31522c04c5d33fa63a80>
>    Changed characters:
>    - cbb3fbd
>    <https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1/commits/cbb3fbd5473609843b41054d725e27d55e3f2d17>
>    Update README.md
>
> File Changes
>
> (1 file
> <https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1/files>
> )
>
>    - *M* web-passwordgenerator-project/index.js
>    <https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1/files#diff-7c3f08aee0b2912100859138ee1bc751a8aad1c2ba63eae184af8051df50fe63>
>    (3)
>
> Patch Links:
>
>    -
>    https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1.patch
>    -
>    https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ravikishore23/Web-Random-Password-Generator/pull/1>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFNZCQTJURAT5M6G3HBBV7323PIB7AVCNFSM6AAAAAB35VXM2OVHI2DSMVQWIX3LMV43ASLTON2WKOZTGAZDEMRRGE2TSMY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

MrKumaran committed:
Added:
Title color change
changes:
Copy function

MrKumaran committed:
Chnaged title class to id

MrKumaran committed:
Changed background and removed repetations

MrKumaran commented:
Yooo! Another pull request,
Added new function to change color of "random generator" for every one second.
changed background to give somewhat natural look and feel(atleast for me)
removed repetitions in css

MrKumaran committed:
Making sure character won't repeat more than 2 times

MrKumaran commented:
new commit: Making sure character won't repeat more than 2 times


review code and accept pull

MrKumaran commented:
CSS need more refactor ig but it works so leaving most part as it
